src/main/scala/com/typesafe/sbt/packager/rpm/RpmMetadata.scala

```diff
@@ -4,6 +4,8 @@ package rpm
 
 import com.typesafe.sbt.packager.linux.{ LinuxPlugin, LinuxPackageMapping, LinuxFileMetaData, LinuxSymlink }
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/1559303177/issues/source?bid=2040925&fileBranchId=2489531#l5)